### DISCUSSION TITLE
Logout

### DISCUSSION THREAD

**Discussion by dieBakterie** [prasanthrangan/hyprdots#1191] [Q&A]
Is it normal that it takes over a minute after logout `SUPER + BACK` to take effect? I've an Nvidia GPU. Ok it seems like my running music was the reason for IT to take so long but not sure

**Comment by kRHYME7** [comment]
try pressing any key on the keyboard

**Reply by dieBakterie** [reply]
Is this intended to Work this way? So a keypress triggers IT?

**Reply by Enigma1309** [reply]
probably a bug
